libs/api/domains/health-insurance/src/lib/accident-notification.service.ts

```
@@ -55,34 +56,42 @@ const mapConfirmationType = (confirmationTypeId: number | undefined) => {
         }))
         .reduce(
           (prev, curr) => ({ ...prev, ...curr }),
           {},
         ) as AccidentNotificationAttachment,
       receivedConfirmations: accidentStatus.confirmations
         ?.map((x) => ({
-          [mapConfirmationType(x.confirmationType)]: !!x.isReceived,
+          [mapConfirmationType(x.party)]: !!x.received,
         }))
         .reduce(
           (prev, curr) => ({ ...prev, ...curr }),
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)